composer.json

```diff
@@ -64,7 +66,9 @@
             "composer config repositories.account path ../account",
             "composer config repositories.image-resizer path ../image-resizer",
             "composer config repositories.menu path ../menu",
-            "composer require rapidez/core:'dev-master as 0.1' rapidez/account:'dev-master as 0.2' rapidez/image-resizer:'dev-master as 0.1' rapidez/menu:'dev-master as 0.1'"
+            "composer config repositories.wishlist path ../wishlist",
+            "composer config repositories.reviews path ../reviews",
+            "composer require rapidez/core:'dev-master as 0.1' rapidez/account:'dev-master as 0.2' rapidez/image-resizer:'dev-master as 0.1' rapidez/menu:'dev-master as 0.1' rapidez/wishlist:'dev-master as 0.1' rapidez/reviews:'dev-master as 0.1'"
```

Comment:
Is this still up-to-date? The core at 0.1 isn't going to work as some packages require a higher version?